### DISCUSSION TITLE
Batch/Spreadsheet mode?

### DISCUSSION THREAD

**Discussion by abraunstein** [SublerApp/Subler#31] [General]
I have a recurring need to work with moderately large (i.e. numbering in the hundreds or low thousands) of video files where the goal is to update the metadata of the files - preferably without forcing a re-encode as that could take a very long time and introduce quality issues.  The data will be created and edited in a spreadsheet, but can be exported in other formats if necessary.  It seems as if Subler can get me all of the metadata stamping functionality I need (which includes the iTunes-specific tags), but as far as I can tell, the batch mode that it supports is more about bulk conversion and that any metadata changes are introduced via pulling from public sources.  Does anybody know of a means by which my use case could be addressed?  If there is no feature that allows for this kind of manipulation, is it scriptable via the command line?

**Comment by Jon-G-Max** [comment]
Yes, look for sublercli. you could probably build a script using the spreadsheet as data to iterate through your movies.  Look here...

https://bitbucket.org/galad87/sublercli/downloads/

**Comment by abraunstein** [comment]
Thank you for the pointer!  I'll give it a look this weekend and see how it goes.
